rfc/0001_trace_parent.md

```diff
@@ -0,0 +1,85 @@
+# 0001: Trace-Parent Accessors
```

Comment:
@yurishkuro @wu-sheng responding to the name. I'm happy to change it, but "SpanContext Accessors" sounds a little vague. The proposal is specific to Trace-Parent, so perhaps "SpanContext accessors for Trace-Parent fields" would work?